u2f-gae-demo/war/js/u2f-api.js

```diff
@@ -168,6 +180,17 @@ u2f.getChromeRuntimePort_ = function(callback) {
 };
 
 /**
+ * Return a 'port' abstraction to the Authenticator app.
+ * @param {function(u2f.WrappedChromeRuntimePort_)} callback
+ * @private
+ */
+u2f.getAuthenticatorPort_ = function(callback) {
+  setTimeout(function() {
```

Comment:
It's just convention (see the constructor for the Chrome port), the idea being that if the constructor is changed in the future to be blocking, then the callers, like the one here, will just work.

Though in this case, the page's JS should have already used setTimeout.